ngraph/core/include/ngraph/op/tensor_iterator.hpp

```diff
@@ -30,9 +30,9 @@ namespace ngraph
                 std::shared_ptr<Node>
                     clone_with_new_inputs(const OutputVector& new_args) const override;
                 /// \return the body of the iteration
-                std::shared_ptr<Function> get_body() const { return m_body; }
+                std::shared_ptr<Function> get_body() const { return m_bodies[0]; }
```

Comment:
Maybe we can add an assert that number of bodies is equal to 1

Reply:
Checking number of bodies and input descriptis is located in validate_and_infer_type function